examples-gallery/plot_crane_moving_a_load.py

```diff
@@ -0,0 +1,294 @@
+# %%
+"""
+Crane moving a load
+===================
+
+A load is moved by a crane. The load is rotably connected to the crane by a
+massless rod. The goal is to move the load from the initial position to the
+final position in the shortest possible time. The load must not 'overswing'
+the final position. The load is moved by a force applied to its
+suspension point. The jib of the crane is extended in the horizontal
+X direction.
+
+**Constants**
+
+- l: length of the rod attaching the load to the crane [m]
+- m1: mass of mover attached to the arm of the crane [kg]
+- m2: mass of the load [kg]
+- g: acceleration due to gravity [m/s²]
+
+**States**
+
+- xc: x-coordinate of the mover [m]
+- xl: x-coordinate of the load [m]  dependent coordinate
+- yl: y-coordinate of the load [m]  dependent coordinate
+- q: angle of the rod [rad]
+- uxc: velocity of the mover in x-direction [m/s]
+- uxl: velocity of the load in x-direction [m/s] dependent speed
+- uyl: velocity of the load in y-direction [m/s] dependent speed
+- u: angular velocity of the rod [rad/s]
+
+**Specifieds**
+
+- F: force applied to the mover [N]
+
+"""
+import sympy.physics.mechanics as me
+from collections import OrderedDict
```

Comment:
OrderedDict is no longer needed, it is a remnant of Python 2 not ordering dictionaries by default.